### PR TITLE
Update tutorial and navbar items

### PR DESCRIPTION
**Pull request by churichard**
- The tutorial now mentions external credits
- The GitHub and Print buttons are now icons
- The tutorial icon has been moved into the dropdown

### CODE REVIEW

File: tigerpath/static/app/js/tutorial.js [hunk 1]
```diff
@@ -26,17 +28,19 @@ function startIntro() {
...
+        element: '#main-view-tabs',
+        intro:
+          'If you have AP credits, summer courses, or other external credits you want to show up in your requirements, you can switch tabs here and add them.',
       },
       {
-        element: '#help-btn',
-        intro: "If you've missed anything, feel free to view this tutorial at any time by clicking here. We hope you enjoy using TigerPath!"
+        element: '#netid-btn',
+        intro:
+          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial here at any time.<br><br>We hope you enjoy using TigerPath!",
```
Comment (bnehoran) [inline tigerpath/static/app/js/tutorial.js:46]:
```suggestion
          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial here again at any time.<br><br>We hope you enjoy using TigerPath!",
```

File: tigerpath/static/app/js/tutorial.js [hunk 2]
```diff
@@ -26,17 +28,19 @@ function startIntro() {
...
-        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`
+        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`,
       },
       {
-        element: '#netid-btn',
-        intro: "If you would like to change your major, add a nickname, or modify other settings, you can click here and then on \"Settings\"."
+        element: '#main-view-tabs',
+        intro:
+          'If you have AP credits, summer courses, or other external credits you want to show up in your requirements, you can switch tabs here and add them.',
       },
```
Comment (bnehoran) [inline tigerpath/static/app/js/tutorial.js:37]: Could you maybe add step for the print button?
It would say something like along the lines of "Once you are happy with the courses you've added, you can print or export your schedule here to share with your advisors."

File: tigerpath/static/app/js/tutorial.js (hunk 1 above)
Comment (churichard) [reply tigerpath/static/app/js/tutorial.js:46]: "here again" sounds a bit weird to me. Is it really necessary to add "again"? I think most users would know they're looking at the tutorial, or they can infer it from the "if you've missed anything". And it doesn't seem like a big deal to me anyway if they view the tutorial and realize they've already seen it

Comment (bnehoran) [reply tigerpath/static/app/js/tutorial.js:46]: Yeah, it's obviously not a big deal. Just thought it's slightly clearer.
If you don't like the "here again" wording you could also do
```suggestion
          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial again here at any time.<br><br>We hope you enjoy using TigerPath!",
```
Not a big deal either way.

**Review by bnehoran — APPROVED** [review]

File: tigerpath/static/app/js/tutorial.js (hunk 1 above)
Comment (churichard) [reply tigerpath/static/app/js/tutorial.js:46]: Still seems a bit wordy to me. I can't find a way to formulate it to flow nicely and it doesn't seem like an incredibly important piece of information to add, so I'll just leave it as is
